common/model-views.h

```diff
@@ -688,7 +689,8 @@ namespace rs2
         frame_queues queues;
         std::mutex _queue_lock;
         bool _options_invalidated = false;
-        int next_option = RS2_OPTION_COUNT;
+        int next_option = 0;
```

Comment:
Can be removed?

Reply:
Why do you think we need to remove it? its indicates the next option we will need to update in next iteration